scopes/webpack/webpack/webpack.bundler.ts

```diff
@@ -16,11 +16,13 @@ export class WebpackBundler implements Bundler {
      */
     private configs: Configuration[],
 
-    private logger: Logger
+    private logger: Logger,
+
+    private webpack
   ) {}
 
   async run(): Promise<BundlerResult[]> {
-    const compilers = this.configs.map((config) => webpack(config as any));
+    const compilers = this.configs.map((config: any) => this.webpack(config));
```

Comment:
don't we want to have a fallback to the webpack as it was before in case it wasn't provided?

Reply:
just saw you passed it from the webpack.main.runtime so maybe you can ignore it.